Makefile

```diff
@@ -164,7 +164,7 @@ build/$(CONVERTER_JAR): $(CONVERTER_SOURCES) $(RESOURCES)
 	$(RM) -r build/converter
 
 %.class: %.java
-	$(JAVAC) -source 7 -target 7 -Xlint:-options -g:none $^
+	$(JAVAC) -source ${TARGET_RELEASE} -target ${TARGET_RELEASE} -Xlint:-options -g:none $^
```

Comment:
I don't see where TARGET_RELEASE is defined.

Reply:
I replaced it `JAVA_TARGET`